tests/components/airly/test_diagnostics.py

```diff
@@ -16,30 +15,11 @@ async def test_entry_diagnostics(
     hass: HomeAssistant,
     aioclient_mock: AiohttpClientMocker,
     hass_client: ClientSessionGenerator,
+    snapshot: SnapshotAssertion,
 ) -> None:
     """Test config entry diagnostics."""
     entry = await init_integration(hass, aioclient_mock)
 
-    coordinator_data = json.loads(load_fixture("diagnostics_data.json", "airly"))
```

Comment:
`diagnostics_data.json` is now unused and thus this fixture could be cleaned up, right?